3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,226 @@
+// Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+
+const indexToRemove = names.indexOf(nameToRemove)
+if (indexToRemove !== -1) {
+    names.splice(indexToRemove, 1)
+}
+
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+
+
+
+
+// When will we be there??
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function calculateTravelTime(travelInformation) {
+    const totalTravelTime = travelInformation.destinationDistance / travelInformation.speed;
+    const hours = Math.floor(totalTravelTime);
+    const minute = Math.round((totalTravelTime - hours) * 60)
+    return `${hours} ours and ${minute} minutes`
+}
+
+
+const travelTime = calculateTravelTime(travelInformation);
+console.log(travelTime); // 8 hours and 38 minutes
+
```

Comment:
🎊 Well done! Your approach to the problem is straightforward and gets the job done. You've made good use of basic JavaScript constructs to arrive at the solution. ✅
❌ **Variable Naming:** While `minute` works, it's a singular term, which may be confusing when the value can be more than one. Consider renaming it to minutes for clarity.